starters/next-wordpress-starter/components/post.jsx

```diff
@@ -5,19 +5,16 @@ export default function Post({
   post: { title, date, featuredImage, content },
 }) {
   return (
-    <article className="prose lg:prose-xl mt-10 mx-auto">
+    <article className="prose lg:prose-xl mt-10 mx-auto max-w-screen-lg p-4">
       <h1>{title}</h1>
       <p className="text-sm text-gray-600">{new Date(date).toDateString()}</p>
 
       <Link passHref href="/">
         <a className="font-normal">Home &rarr;</a>
       </Link>
-      <div className="mt-12 max-w-lg mx-auto lg:grid-cols-3 lg:max-w-screen-lg">
-        {featuredImage && (
-          <div
-            className="relative w-full rounded-lg shadow-lg overflow-hidden mb-10"
-            style={{ height: "50vh" }}
-          >
+      {featuredImage && (
```

Comment:
I am not able to see images after I pulled the latest changes.
Can we revert this section to how it was? If you want to put that outer div inside of the conditional too, that's OK.

```jsx
      <div className="mt-12 max-w-lg mx-auto lg:max-w-screen-lg">
        {featuredImage && (
          <div
            className="relative w-full rounded-lg shadow-lg overflow-hidden mb-10"
            style={{ height: "50vh" }}
          >
            <Image
              priority
              src={featuredImage.node.sourceUrl}
              layout="fill"
              objectFit="cover"
              alt={featuredImage.node.alt || "Featured Image"}
            />
          </div>
        )}
      </div>
```